File: first-unique-character-in-a-string/first-unique-character-in-a-string.go
func firstUniqChar(s string) int {
    n := len(s)
    arr := [26]int{}
    for i:=0; i<n; i++{
        arr[s[i]-97] += 1
    }
    for i := 0; i<n; i++{
        if arr[s[i]-97] == 1{
            return i
        }
    }
    return -1
}

File: first-unique-character-in-a-string/first-unique-character-in-a-string.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        
        count = [0]*26
        for i in range(len(s)):
            count[ord(s[i]) - ord('a')] += 1
        for i in range(len(s)):
            if count[ord(s[i]) - ord('a')] == 1:
                return i
        return -1
            